src/liquid/components/ld-radio/ld-radio.tsx

```diff
@@ -146,35 +181,52 @@ export class LdRadio implements InnerFocusable {
   }
 
   componentWillLoad() {
-    if (this.el.closest('form')) {
-      this.hiddenInput = document.createElement('input')
-      this.hiddenInput.required = this.required
-      this.hiddenInput.type = 'radio'
-      this.hiddenInput.style.visibility = 'hidden'
-      this.hiddenInput.style.position = 'absolute'
-      this.hiddenInput.style.pointerEvents = 'none'
-      this.hiddenInput.checked = this.checked
+    const outerForm = this.el.closest('form')
 
-      if (this.value) {
-        this.hiddenInput.value = this.value
-      }
+    if (outerForm && !this.autocomplete) {
+      this.autocomplete = outerForm.getAttribute('autocomplete')
+    }
 
+    if (outerForm || this.form) {
       if (this.name) {
```

Comment:
Same as for checkbox.
```suggestion
    if (this.name && (outerForm || this.form)) {
```